### DISCUSSION TITLE
EL5 Feedback

### DISCUSSION THREAD

**Discussion by siwelwerd** [TeamBasedInquiryLearning/precalculus#221] [Show and tell]
https://teambasedinquirylearning.github.io/precalculus/2024/EL5.html

Author: @tdegeorge 
Reviewer: @kathypinzon 

This one is already ready for review @kathypinzon

**Comment by kathypinzon** [comment]
Throughout use \cdot instead of the *

**Comment by kathypinzon** [comment]
Maybe before Activity 5.5.2 as a remark or in the intro put the things they need to recall and use.  Log_b M=log_b N if and only if M=N., exponentials and logs are inverses so log_b(b^{k})=k, etc....
Then,
According to the law of exponents, we know that ... Consider this equation as you move through this activity.
reword 5.5.2 part (a) - Let a=10^x and b=10^y.  How could you rewrite the left side of the equation 10^x \cdot 10^y?
5.5.2(b) I would say - Recall from Section EL3 that Log_b M=log_b N if and only if M=N.  Use this property to take apply the logarithm to both sides of the rewritten equation from 5.5.2(a).
5.5.2(e) I don't think you should put the whole equation there, just the right sides.
Then, I think you need one more summary step asking them what the final transformed property looks like

**Comment by kathypinzon** [comment]
Model 5.5.3 after suggestions to 5.5.2 above.

**Comment by kathypinzon** [comment]
I would reverse remark 5.5.5 and definition 5.5.4.  In remark 5.5.5 don't say derive - maybe something like - you explored an example of this property since they only used base 10...

**Comment by kathypinzon** [comment]
5.5.6 (b) Drew would say you should put the answers from this part in the same order as the corresponding answers in part a.

**Comment by kathypinzon** [comment]
You might want to add something to the directions to explain what expanded fully means.
5.5.8 (c) answer should be "D" is also correct, but it is not expanded fully.

**Comment by kathypinzon** [comment]
5.5.9 update directions to say "condense into a single logarithm"
(a) answer could say "B is also correct, but it is not condensed fully."

**Comment by kathypinzon** [comment]
5.5.12 I would rewrite one of them with ln instead of both with log.
